src/screens/tenants/CreateTenantDialog.tsx

```diff
@@ -46,24 +47,27 @@ export default class CreateTenantDialog extends React.Component<Props, State> {
 
   public constructor(props: Props) {
     super(props);
-    this.tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS;
+    let tenantEndpointClouds: EndpointCloud[];
+    if (__DEV__) {
+      tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS_DEV;
+    } else {
+      tenantEndpointClouds = Configuration.ENDPOINT_CLOUDS;
+    }
+    const newTenantEndpointCloudId = tenantEndpointClouds.find((tenantEndpointCloud) => tenantEndpointCloud.id === 'scp')?.id;
```

Comment:
I do not think the filtering on 'scp' is necessary. And you like to put the result of helpers in intermediate variables ;) I'm not very fond of it, I do not think that makes the code easier to read, especially with JS or TS that permits to use arrow function and the like to directly use treatment result. That's just nitpicking ;)